docs/user_manual/general_navigation.md

```diff
@@ -5,8 +5,15 @@ section: 2
 ---
 
 
-
 {% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Home Screen" %}
```

Comment:
Needs added to the table of contents as well

Reply:
Done